libctru/include/3ds/services/ac.h

```diff
@@ -90,3 +95,36 @@ Result ACU_GetLastErrorCode(u32* errorCode);
  * @param errorCode Pointer to output the error code to.
  */
 Result ACU_GetLastDetailErrorCode(u32* errorCode);
+
+/**
+ * @brief Prepares a buffer to hold the configuration data to start a connection.
+ * @param config Pointer to an acuConfig struct to contain the data.
+ */
+Result ACU_CreateDefaultConfig(acuConfig* config);
+
+/**
+ * @brief Sets something that makes the connection reliable.
+ * @param config Pointer to an acuConfig struct used with ACU_CreateDefaultConfig previously.
+ * @param area Always 2 ?
+ */
+Result ACU_SetNetworkArea(acuConfig* config, u8 area);
+
+/**
+ * @brief Sets the slot to use when connecting.
+ * @param config Pointer to an acuConfig struct used with ACU_CreateDefaultConfig previously.
+ * @param type Allowed slots flag. 1 for slot 1, 2 for slot 2, 4 for slot 3.
```

Comment:
If it's a bitmask, use BIT(N) for this comment?

Reply:
seems to be, the AddAllowApType and AddDenyApType do bitwise operations on some part of the config buffer, should they be added?

Reply:
sure